alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Utxo.hs

```diff
@@ -265,8 +268,14 @@ feesOK ::
     HasField "_minfeeA" (Core.PParams era) Natural,
     HasField "_minfeeB" (Core.PParams era) Natural,
     HasField "_prices" (Core.PParams era) Prices,
-    HasField "_collateralPercentage" (Core.PParams era) Natural,
-    HasField "address" (Alonzo.TxOut era) (Addr (Crypto era))
+    Core.Script era ~ Script era,
+    Core.TxBody era ~ TxBody era,
+    Show (Core.Value era),
+    PrettyA (Core.Value era),
+    PrettyA (Core.PParamsDelta era),
+    PrettyA (Core.AuxiliaryData era),
```

Comment:
maybe remove the pretty printing constraints?

Reply:
done